### PR TITLE
Style check of SLE RT Hardware Testing

### PR DESCRIPTION
**Pull request by tomschr**
* add `<date>`
* Slightly rephrased overly long sentences

@JeffreyCheung: I run our style checker and it found several minor things. Could you look at the changed sentences and give me feedback if it's still ok? Thanks!

### CODE REVIEW

**Comment by JeffreyCheung** [comment]
I am OK.

File: xml/art_slert_hw_testing.xml [hunk 1]
```diff
@@ -93,9 +94,9 @@
         linkend="sec.hwtest.cyct"/> for details).
       </para>
       <para>
-       In a typical situation, when &productname; works without any problems,
-       the latencies measured with <command>cyclictest</command> should not
-       be very different from those measured by the hardware latency
+       In a typical situation<!--, when &productname; works without any problems,-->
+       the latencies measured with <command>cyclictest</command> should be
+       mostly the same from those measured by the hardware latency
```
Comment (lpechacek) [inline xml/art_slert_hw_testing.xml]: +1

File: xml/art_slert_hw_testing.xml [hunk 2]
```diff
@@ -179,17 +179,17 @@
...
       interrupts, SMIs). </para>
 
-    <para> The hardware latency detector module works by hogging all of
-      the CPUs for configurable amounts of time (by calling
-        <systemitem class="other" otherclass="function"
-       >stop_machine()</systemitem>), polling the CPU Time
-      Stamp Counter (TSC) for some period and then looking for gaps in the TSC
-      data. These gaps indicate times when the polling was interrupted
-      because of SMIs. </para>
+    <para> The hardware latency detector module works by demanding all of
```
Comment (lpechacek) [inline xml/art_slert_hw_testing.xml]: "... works by making all CPUs busy for ..."?

File: xml/art_slert_hw_testing.xml [hunk 3]
```diff
@@ -179,17 +179,17 @@
...
+      the CPUs for configurable amounts of time. This is done by calling
+      <systemitem class="other" otherclass="function"
+       >stop_machine()</systemitem>, polling the CPU Time Stamp Counter (TSC)
+      for some period, and then looking for gaps in the TSC data.
+      These gaps indicate times when the polling was interrupted
+      because of SMIs.</para>
```
Comment (lpechacek) [inline xml/art_slert_hw_testing.xml]: maybe "... the polling was disturbed by System Management Interrupts (SMI)."

File: xml/art_slert_hw_testing.xml (hunk 2 above)
Comment (tomschr) [inline xml/art_slert_hw_testing.xml]: This is much better! I'll take it! ;-)

File: xml/art_slert_hw_testing.xml (hunk 3 above)
Comment (tomschr) [inline xml/art_slert_hw_testing.xml]: Yes, this is much better. Integrated.

**Comment by tomschr** [comment]
Integrated your corrections and merged. Thanks! 👍

**Pull request closed without merging**